rest-server/src/routes/job.js

```diff
@@ -31,14 +31,14 @@ router.route('/')
     .get(param.jobQuery, jobController.list)
 
     /** POST /api/v1/jobs - Update job */
-    .post(tokenConfig.check, param.validate(jobConfig.schema), jobController.init, jobController.update);
+    .post(tokenConfig.check, param.validate(jobConfig.schema), param.checkKillAllOnCompletedTaskNumber, jobController.init, jobController.update);
```

Comment:
>checkKillAllOnCompletedTaskNumber [](start = 69, length = 33)

why don't we put checkKillAllOnCompletedTaskNumber into param.validate?

Reply:
`param.validate` is a general function which checks a json using a schema, it doesn't limit to job configuration

Reply:
it doens't seem to scale by defining a new function to check just one value

---
In reply to: [191151374](https://github.com/Microsoft/pai/pull/625#discussion_r191151374) [](ancestors = 191151374)